core/domain/question_domain.py

```diff
@@ -48,6 +48,8 @@
 QUESTION_PROPERTY_LANGUAGE_CODE = 'language_code'
 QUESTION_PROPERTY_QUESTION_STATE_DATA = 'question_state_data'
 QUESTION_PROPERTY_LINKED_SKILL_IDS = 'linked_skill_ids'
+QUESTION_PROPERTY_NOT_APPLICABLE_MISCONCEPTION_IDS = (
```

Comment:
Nit: NOT should be NON, here and elsewhere.

(or, perhaps better -- "inapplicable")

Reply:
Done.